java/fury-core/src/main/java/org/apache/fury/resolver/ClassResolver.java

```diff
@@ -160,45 +160,45 @@ public class ClassResolver {
   // bit 0 unset indicates class is written as an id.
   public static final byte USE_CLASS_VALUE_FLAG = 0b1;
   // preserve 0 as flag for class id not set in ClassInfo`
-  public static final short NO_CLASS_ID = (short) 0;
-  public static final short LAMBDA_STUB_ID = 1;
```

Comment:
Different languages have different type systems. The type id for java native serialization and xlang serialization may be different.

Reply:
Ok, I see, so we need to define a new set of TypeId in java for xlang spec?